tests/block.js

```diff
@@ -7,11 +7,24 @@ const Block = require('../index.js')
 
 tape('[Block]: block functions', function (t) {
   const testData = require('./testdata.json')
-
-  t.test('should test validateTransactions', function (st) {
-    var block = new Block(Buffer.from(testData.blocks[0].rlp.slice(2), 'hex'))
+  
+  function testTransactionValidation(st, block) {
     st.equal(block.validateTransactions(), true)
-    st.end()
+    
+    block.genTxTrie(function() {
+      st.equal(block.validateTransactionsTrie(), true)
+      st.end()
```

Comment:
Since this is now in callback I think you need to add `st.expect(2)` or something like that to the outside so it knows 2 checks are required.